bootstrap/bootstrap.go

```diff
@@ -120,6 +121,10 @@ func RunAndReturnWaitGroup(
 		if err != nil {
 			fatalError(fmt.Errorf("failed to create SecretProvider: %s", err.Error()), lc)
 		}
+
+		// Bypass the zero trust zitidfied transport for Core Keeper Configuration client
```

Comment:
@cloudxxx8 / @jackchenjc, this has an unintentional side-effect of breaking device-virtual. Strictly speaking this also should not be necessary as the zitified transport is configured with fallback, meaning if it can't dial a zitified endpoint, it should fallback to using an underlay dialer.

I'm sure this was a problem somewhere/somehow, but can I work with you to figure out why??